hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -3366,3 +3376,64 @@ func (r *NodePoolReconciler) reconcileMirroredConfigs(ctx context.Context, logr
 	}
 	return nil
 }
+
+// SetPerformanceProfileConditions checks for performance profile status updates, and reflects them in the nodepool status conditions
+func (r *NodePoolReconciler) SetPerformanceProfileConditions(ctx context.Context, logger logr.Logger, nodePool *hyperv1.NodePool, controlPlaneNamespace string, toDelete bool) error {
+	if toDelete {
+		ppConds := []string{
+			hyperv1.NodePoolPerformanceProfileTuningAvailableConditionType,
+			hyperv1.NodePoolPerformanceProfileTuningProgressingConditionType,
+			hyperv1.NodePoolPerformanceProfileTuningUpgradeableConditionType,
+			hyperv1.NodePoolPerformanceProfileTuningDegradedConditionType,
+		}
+		for _, ppCond := range ppConds {
```

Comment:
could you avoid shortcuts like ppCond or ppConds? I would change by something like condition and performanceConditions